website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -77,3 +77,17 @@
         <p data-bind="html: message, attr.class: messageClass"></p>
     </div>
 </div>
+
+ <script>
+    // Make s3 settings variables globally accessible for relevant JS
+    <%
+        import json
+        from website.addons.s3.settings import DEFAULT_BUCKET_LOCATION
+    %>
+    window.contextVars = $.extend(true, {}, window.contextVars, {
+        s3Settings: {
+            defaultBucketLocationValue: ${json.dumps(DEFAULT_BUCKET_LOCATION['value'])},
```

Comment:
The preferred way to render python strings into JS in mako is to use the `sjson , n` filter. See https://github.com/CenterForOpenScience/osf.io/pull/2917#issuecomment-119656823